ignore case on username login release note (#1788)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-182-0-ignore-case-on-username-login.md

Reviewer: This Feature not released Yet!! This is still being worked on.

Reviewer 2: > This Feature not released Yet!! This is still being worked on.

Without a date it will not go live anyway. But it is strange that it has a version then?

Reviewer: yes! there are some bugs identified after initial implementation. Michal is working on it.

Author: Michał inform me that all bugs are fixed - and we can merge it

Reviewer 2: > Michał inform me that all bugs are fixed - and we can merge it

What is the correct fix version then?

---
date: ""

> title: Ignore Case on Username Login

Reviewer 2: Option to ignore case of username or alias on login

> title: Ignore Case on Username Login

Reviewer 2: ```suggestion
title: Option to ignore case of username or alias on login
```

Reviewer 3: if we ignore case on username and alias both please mention it in title.

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-55565
version: 10.20.182.0
---

> The new solution has been introduced to simplify the login process, enabling tenant administrators to activate the option of allowing users to log in with case-insensitive usernames or aliases.

Reviewer 2: ```suggestion
To simplify the login process, tenant administrators can now activate an option which allows users to log in with case-insensitive usernames or aliases.
```
